docs/rules/function-paren-newline.md

```diff
@@ -14,6 +14,7 @@ This rule has a single option, which can either be a string or an object.
 * `"never"` disallows line breaks inside all function parentheses.
 * `"multiline"` (default) requires linebreaks inside function parentheses if any of the parameters/arguments have a line break between them. Otherwise, it disallows linebreaks.
 * `"consistent"` requires consistent usage of linebreaks for each pair of parentheses. It reports an error if one parenthesis in the pair has a linebreak inside it and the other parenthesis does not.
+* `"consistent-arguments"` requires consistent usage of linebreaks for each pair of parentheses and parameters/arguments. It allows linebreaks inside function parentheses if there is only one parameter/argument.
```

Comment:
From reading this documentation, I'm confused about what this option is supposed to do when there is more than one argument. Could you add a sentence or two to describe what the option does in that case? The description here is currently incomplete because it only describes the one-argument case. (This also makes it difficult to review the implementation because it's not clear what the intended behavior is.)

Is the intention that this is the same as the `multiline` option except that it tolerates linebreaks when a single argument is present? If so, maybe it would be better to call this option something like `multiline-arguments` rather than `consistent-arguments`. If my understanding is correct, it doesn't seem like this is really related to the `consistent` option.

Reply:
I agree. Fixed.